specifyweb/frontend/js_src/lib/components/WbActions/WbResults.tsx

```diff
@@ -11,17 +12,18 @@ export function WbResults({
   readonly hasUnsavedChanges: boolean;
   readonly onToggleResults: () => void;
 }): JSX.Element {
-
   return (
     <>
-      <Button.Small
-        aria-haspopup="tree"
-        disabled={hasUnsavedChanges}
-        title={hasUnsavedChanges ? wbText.wbUploadedUnavailable() : ''}
-        onClick={handleToggleResults}
-      >
-        {commonText.results()}
-      </Button.Small>
+      <ErrorBoundary dismissible>
```

Comment:
it's best to wrap a call to a component in a react boundary, rather than call react boundary inside the component
i.e do `<ErrorBoundary><WbResults/></ErrorBoundary>`
this way, if there are exceptions in the WbResults function, those won't crash the entire workbench

the way you have it here right now, only exceptions in the components that WbResults renders would be isolated - not good enough

could you please fix that here and in the other places where you have ErrorBoundaries?

sorry, I should have been more clear when I told you to add them